latest/index.bs

```diff
@@ -820,6 +820,16 @@ Version History {#history}
       <td>Description</td>
     </tr>
   </thead>
+  <tr>
+    <td>0.4.0</td>
+    <td>2022-02-08</td>
+    <td>multiscales: add axes type, units and coordinateTransformations</td>
+  </tr>
+  <tr>
+    <td>0.4.0</td>
```

Comment:
I'd be for combining the two rows.

Reply:
Ah, I see @constantinpape was for the opposite. Hmm.... happy to look into it later. (Also need to look into removing the "W3C" in the versioned pages)

Reply:
I don't have a strong opinion here. Maybe we could start to introduce sub-points? like
```
- v0.4
  - multiscales: ...
  - high-content-screen: ...
```

Reply:
FWIW, I am not the biggest fan of the double row either but I was hoping someone would come with an authoritative suggestion 😄 

In the absence of a strong consensus, I'd be inclined to not delay the publication of the spec and capture this as an issue to investigate. We will certainly face similar cases in the future and it would be good not to be constrained in terms of what we want to document.